bitnami/nginx/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: nginx
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/nginx
-version: 16.0.4
+version: 16.0.5
```

Bumping chart version because 16.0.5 has already been released https://github.com/bitnami/charts/blob/35ef5e68076a9e7e43b36c3d8e5d9589946b88b6/bitnami/nginx/Chart.yaml#L37

```suggestion
version: 16.0.6
```